### ISSUE TITLE
[Question] H1 Arm Control

### ISSUE THREAD

**Issue by RiccardoZuppetti** [isaac-sim/IsaacLab#1319]
### Question

Hi,

Based on the current policy that allows to H1 to move in the surrouding environment, how would it be possible to control the movement of the humanoid's arms, while maintaining balance? Probably the policy in question does not take arm control into account.

**Comment by jsmith-bdai** [comment]
That's correct, the current policy doesn't take arm control into account. 

You could create a new task and train it to control the arms, or modify the actions being sent to the arm's actuators from the trained policy (but this might not be robust)

**Comment by RiccardoZuppetti** [comment]
> That's correct, the current policy doesn't take arm control into account.

Actually, I'm exploiting the same example built-in within Isaac Sim, this [one](https://docs.omniverse.nvidia.com/isaacsim/latest/isaac_lab_tutorials/tutorial_policy_deployment.html#unitree-h1-humanoid-example)

> You could create a new task and train it to control the arms, or modify the actions being sent to the arm's actuators from the trained policy (but this might not be robust)

How can I create a new task? Is there some reference to which I can refer? I've seen that for the arm control there are some tasks but are mostly for manipulators.

**Comment by jsmith-bdai** [comment]
Yes, we have tutorials for creating tasks [here](https://isaac-sim.github.io/IsaacLab/main/source/tutorials/03_envs/create_manager_rl_env.html), although they are for simpler tasks. 

The H1 task was trained via https://github.com/isaac-sim/IsaacLab/blob/main/source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/h1/flat_env_cfg.py I believe so that would be a good starting point to build off of

**Comment by RiccardoZuppetti** [comment]
Once that a task has been trained, how can I exploit the generated policy in order to create my custom environment in IsaacSim? Is there something that could help me in this sense?
